tests/test-3.6.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Do we need the file is test is commented?

Reply:
I 'll remove it.